src/parser/GenericParser.test.ts

```diff
@@ -674,4 +675,33 @@ describe("GenericParser", () => {
             expect(couplingResult).toMatchSnapshot();
         }, 10000);
     });
+
+    describe("Include files with unknown or no file extension", () => {
+        it("should list files with unknown file extension", async () => {
+            const inputPath = fs.realpathSync(unknownTestResourcesPath);
+            const parser = new GenericParser(getParserConfiguration(inputPath));
+            const results = await parser.calculateMetrics();
+
+            const filePath = fs.realpathSync(unknownTestResourcesPath + "example.unknownExtension");
+            expect(results.unknownFiles.includes(filePath)).toBe(true);
+        });
+
+        it("should list files with no file extension", async () => {
```

Comment:
this seems to be the identical test as the test above

Reply:
Fixed